ultra/ultra/env/ultra_env.py

```diff
@@ -148,41 +140,46 @@ def reset(self):
 
         return observations
 
-    def get_scenarios(self, scenario_info: Tuple[str, str]) -> List[str]:
-        """Finds all of the scenarios from the given scenario information
-
-        To obtain the path of scenarios:
-            1. Get the filename patterns of the train and test scenarios from
-               ultra/config.yaml from the given task and level.
-            2. Change the filename patterns from relative to absolute path
-            3. Depending on the evaluation mode, we can provide glob.glob with
-               the appropriate filename pattern to return a list of scenario dirs
+    @staticmethod
+    def get_scenarios_from_scenario_info(
+        scenario_info: Union[Tuple[str, str], Sequence[str]], eval_mode: bool
+    ) -> List[str]:
```

Comment:
I was thinking that we should make the `eval_mode` a keyword argument because it is not necessary when there are explicit scenarios passed in. And also if we are planning to use it as a static method, it will be better to inform that the second argument is indeed the evaluation mode param

Reply:
Ahh yes, thanks, good call.